docs/src/routes/docs/[...3]modules/transaction-preview.md

```diff
@@ -73,24 +72,146 @@ const onboard = Onboard({
 ```
 
+### Standalone Usage
+
+
+To use the Transaction Preview package without web3-onboard all a developer needs to do is: 
+- Execute the entry function from the `@web3-onboard/transaction-preview` package and optional params
+- Run the returned `init` function with their [Blocknative API key](https://onboard.blocknative.com/docs/overview/introduction#optional-use-an-api-key-to-fetch-real-time-transaction-data-balances-gas), an initialized instance of their [Blocknative SDK](https://www.npmjs.com/package/bnc-sdk) and a containerElement string with the html ID of the target element to append the visualization to
+- Finally pass a transaction meant for a wallet provider (created using libraries like Ethers or Web3)
+
+With the above steps a UI will be rendered with the balance changes and gas used.
+```typescript
+import transactionPreviewModule from '@web3-onboard/transaction-preview'
+
+const {init, previewTransaction} = transactionPreviewModule({
+  // Optional: Require balance change approval prior to sending transaction to wallet
+  // Defaults to true
+  // requireTransactionApproval?: false
+
+  //  i18n?: i18nOptions - Internationalization options
+})
+await init({
+/**
+ * Blocknative API key (https://explorer.blocknative.com/account)
+ */
+apiKey: string
+/**
+ * Your Blocknative SDK instance
+ * */
+sdk: SDK
+/**
+ * Optional dom query string to mount UI to
+ * */
+containerElement: string})
+
+// Transaction code here using Ether.js or Web3.js or construct your own transactions
+const simulate = async provider => {
+  const ethersProvider = new ethers.providers.Web3Provider(provider, 'any')
+
+  const signer = ethersProvider.getSigner()
+  const addressFrom = '0xcxxxxxx11111999991111'
+
+  // Uniswap V2
+  const CONTRACT_ADDRESS = '0x7a250d5630b4cf539739df2c5dacb4c659f2488d'
+  const erc20_interface = [
+    'function approve(address _spender, uint256 _value) public returns (bool success)',
+    'function transferFrom(address sender, address recipient, uint256 amount) external returns (bool)',
+    'function balanceOf(address owner) view returns (uint256)'
+  ]
+
+  const uniswapV2router_interface = [
+    'function swapExactTokensForETH(uint amountIn, uint amountOutMin, address[] calldata path, address to, uint deadline) external returns (uint[] memory amounts)'
+  ]
+
+  const weth = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+  const oneInch = '0x111111111117dc0aa78b770fa6a738034120c302'
+  let swapTxData
+  let approveTxData
+  const swapContract = new ethers.Contract(
+    CONTRACT_ADDRESS,
+    uniswapV2router_interface
+  )
+  const erc20_contract = new ethers.Contract(oneInch, erc20_interface)
+  const oneEther = ethers.BigNumber.from('9000000000000000000')
+  approveTxData = await erc20_contract.populateTransaction.approve(
+    CONTRACT_ADDRESS,
+    oneEther
+  )
+
+  const amountOutMin = 0
+  const amountOutMinHex = ethers.BigNumber.from(amountOutMin.toString())._hex
```

Comment:
@Adamj1232 would be better? I would avoid using `_` pre-fixed properties as they are generally considered private and subject to change AFAIK 
```suggestion
  const amountOutMinHex = ethers.BigNumber.from(amountOutMin).toHexString()
```
I think you can remove `toString()` as `BigNumber.from()` accepts both (just tested this)

You could also do this:
```
const amountOutMinHex = '0x0'
```